comps/llms/rayllm/docker/dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest
+
+ENV LANG=en_US.UTF-8
+
+WORKDIR /root/rayllm
```

Comment:
Could we change rayllm to other name so that it will not be confused with RayLLM project?